envs/vagrant/mirrors/group_vars/all.yml

```diff
@@ -10,7 +10,9 @@ percona:
 
 nova:
   scheduler_default_filters: AvailabilityZoneFilter,ComputeFilter
-  novnc_repo: http://172.16.0.18:8098/git/mirror/noVNC.git
+  novnc_repo: http://173.247.104.68:8098/git/mirror/noVNC.git
+  novnc_method: file
+  novnc_url: http://173.247.104.68:8090/files/novnc-0.5.1.tar.gz
```

Comment:
right now that's just a throwaway server for the example mirrors environment.   At some point I'll nuke it and build it correctly and will use a DNS name then.